docs/en/inference/turbomind_config.md

```diff
@@ -96,6 +97,14 @@ The `cache_chunk_size` indicates the size of the k/v cache chunk to be allocated
 - When the value is -1, `cache_max_entry_count` number of k/v cache blocks are allocated.
 - When the value is 0, `sqrt(cache_max_entry_count)` number of k/v cache blocks are allocated.
 
+### prefix caching switch
+
+Prefix caching feature can be controlled by setting the `enable_prefix_caching` parameter. When set to `True`, it indicates that the feature is enabled, and when set to `False`, it indicates that the feature is disabled. The default value is `False`.
```

Comment:
If we default to using True, what are the disadvantages?

Reply:
For scenarios with no common prefix, the prefix matching and caching will bring unnecessary overhead.